### ISSUE TITLE
Error parsing a meta element's content: ';' is not a valid key-value pair separator. Please use ',' instead.

### ISSUE THREAD

**Issue by DiegoJArg** [susam/texme#28]
I think that this line
```
<meta name="viewport" content="width=device-width; initial-scale=1.0">
```
Produces this error in console:
> Error parsing a meta element's content: ';' is not a valid key-value pair separator. Please use ',' instead.

Try to change ';' by ','
```
<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

REFERENCE
https://stackoverflow.com/questions/50713010/webview-is-failing-to-load-some-webpages

**Comment by susam** [comment]
Thank you @DiegoJArg for reporting this issue. This is indeed a bug. It will take me a few days to get to this. In the meantime, a pull request from anyone interested to fix this is welcome.

**Comment by susam** [comment]
Fixed in commit 58c50c4. Fix is available in version 1.2.2. Thanks again for reporting this issue.
